model/build.gradle

```diff
@@ -4,11 +4,10 @@ apply plugin: 'com.google.protobuf'
 protobuf {
   protoc {
     // To build protoc in M1 mac. For context, see: #3912.
-    if (project.rootProject.hasProperty('protobuf_platform')) {
-      artifact = "com.google.protobuf:protoc:3.8.0:${project.rootProject.property("protobuf_platform")}"
-    } else {
-      artifact = "com.google.protobuf:protoc:3.8.0"
-    }
+    Properties properties = new Properties()
+    properties.load(project.rootProject.file("local.properties").newDataInputStream())
```

Comment:
The project doesn't have a local.properties--is this something just on your local machine?

Also, we need to set this up such that it will work for other team members without them needing to set up their own local.properties (otherwise, it's really likely people will get stuck since it won't be clear that they need to do that).

Could we use a different approach?